segmentation_labeling_app/transfers/upload.py

```diff
@@ -0,0 +1,94 @@
+import argschema
+import marshmallow as mm
+import datetime
+import tempfile
+import segmentation_labeling_app.transfers.utils as utils
+
+
+class UploadSchema(argschema.ArgSchema):
+    sqlite_db_file = argschema.fields.InputFile(
+        required=True,
+        description="sqllite input db file")
+    sql_table = argschema.fields.Str(
+        required=False,
+        missing="manifest_table",
+        description="table where the manifests are stored")
+    sql_filter = argschema.fields.Str(
+        required=False,
+        default="",
+        missing="",
+        description="SQL query filter, starting with 'WHERE'")
+    s3_bucket_name = argschema.fields.Str(
+        required=True,
+        description="destination bucket name")
+    contents_prefix = argschema.fields.Str(
+        required=False,
+        default="",
+        missing="",
+        description="key prefix for manifest contents")
+    manifest_prefix = argschema.fields.Str(
+        required=False,
+        allow_none=True,
+        default=None,
+        missing=None,
+        description=("key prefix for manifest. Will default "
+                     "to contents_prefix"))
+    manifest_write_mode = argschema.fields.Str(
```

Comment:
NOTE: as per slack discussion, we probably do not need to support appending a manifest on s3. This option and the supporting code can probably be dropped.

Reply:
removed option to append to existing manifest